table.cc

```diff
@@ -1720,7 +1720,10 @@ seal_snapshot(sstring jsondir) {
 future<> table::snapshot(sstring name) {
```

Comment:
Fixes ????
Explanation?

Reply:
The motivation is described in the pull request message.
I can add a comment here as well.